index.html

```diff
@@ -9,16 +9,75 @@
     <title>Kossi Fioklou Portfolio</title>
 </head>
 <body>
-   <!-- Mobile Menu -->
+  <!--Popup -->
+  <div class="popup-overlay hidden" id="popup">
+    <div class="card popup desk">
+      <div class="card-content desk">
+        <img
+          src="assets/Icon.png"
+          alt="close-overlay"
+          class="popup-close-btn" 
+        />
+        <h2 class="heading-secondary desk" id="name">Tonic</h2>
+        <ul class="card-info desk">
+          <li id="companyName">CANOPY</li>
+          <li>
+            <img
+              src="assets/dot.png"
+              alt="dot"
+              class="card-info-dot"
+            />
+          </li>
+          <li class="input-text" id="job">Back End Dev</li>
+          <li>
+            <img
+              src="assets/dot.png"
+              alt="dot"
+              class="card-info-dot"
+            />
+          </li>
+          <li class="input-text" id="data">2015</li>
+        </ul>
+        <img
+          class="card-img desk"
+          id="img"
+          src="./assets/SnapshootPortfolio.jpg"
+          alt="Tonic"
+        />
+        <div class="desk-pop">
+          <div class="first">
+            <p class="text-secondary mob desk" id="description"></p>
+          </div>
+          <div class="second">
+            <ul class="card-tags desk" id="lang"></ul>
+            <div class="line"></div>
+            <div class="popup-button desk">
+              <a href="#" class="button btn-element desk"
+                >See Live
+                <img src="assets/Icon-1.png" alt="live"
+              /></a>
+              <a href="#" class="button btn-element desk"
+                >See Source
+                <img src="assets/github.jpg" alt="git"
+              /></a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <!--End Popup-->
+
+  <!-- Mobile Menu -->
  <div id="menu" class="mobile-menu">
   <ul >
     <li class="menuList"><a href="#portfolio">Portfolio</a></li>
     <li class="menuList"><a href="#about">About</a></li>
     <li class="menuList"><a href="#contact">Contact</a></li>
   </ul>
   <img src="assets/Icon.png" alt="Close-Menu" class="close" id="close-menu">
-  <div class="mobile-1"></div>
-  <div class="mobile-2"></div>
+  <!--<div class="mobile-1"></div>
+  <div class="mobile-2"></div>-->
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.

I trust you can do this before merging.